gdx/src/com/badlogic/gdx/Files.java

```diff
@@ -82,4 +89,7 @@
 
 	/** Returns true if the local storage is ready for file IO. */

```

Comment:
This JavaDoc could be changed to reflect that it also returns if cache path is available. (The current doc is misleading btw)